test/integration_test.go

```diff
@@ -267,3 +267,22 @@ func TestShowDatabases(t *testing.T) {
 	assert.NoErrorf(t, err, "show databases: %v", err)
 	assert.Equal(t, affected[0].DatabaseTypeName(), "VARCHAR")
 }
+
+func TestDropTable(t *testing.T) {
+	db, err := sql.Open(driverName, dataSourceName)
```

Comment:
By the way, use `t.Skip()` to skip it, because it will drop testing tables.

Reply:
@jjeffcaii fixed